.tasks

```diff
@@ -12,13 +12,24 @@
 # Cockpit bots.
 
 set -ex
+
+# Scan for all tests
 bots/tests-scan
 
+# When run automated, randomize to minimize stampeding hurd
+if [ -t 0 ]; then
+    chance=10
+else
+    chance=$(shuf -i 0-10 -n 1)
+fi
+
 # File issues for these tasks
-bots/po-trigger
-bots/image-trigger
-bots/npm-trigger
-bots/naughty-trigger
+if [ $chance -gt 9 ]; then
```

Comment:
You need `-ge` here, otherwise this will never be true.

Reply:
Are you sure? I think ```shuf -i 1-10``` produces the number 10.

Reply:
Sorry, indeed it does. (That's unlike `random(10)` in most other places).